src/components/ProductCard.tsx

```diff
@@ -10,13 +13,67 @@ type Props = {
 export const ProductCard: FC<Props> = ({ product }) => {
   return (
     <div className={productStyles.card}>
-      <Image
+      <Image 
+        className={productStyles.card__img}
         src={`/${product.image}`}
         alt={product.name}
         width={208}
         height={196}
       />
-      <p>Hello</p>
+
+      <h2 className={`${productStyles.card__title} ${typographyDesktop.bodyText}`}>
+        {product.name}
+      </h2>
+
+      <p className={productStyles.card__price}>
+        <span className={productStyles.card__priceСurrent}
+          >
+          {`$${product.price}`}
```

Comment:
why do you need two symbols of dollar?

Reply:
it`s price first dollar is dollar sign, second is string literal